### ISSUE TITLE
Dashboard showing no data

### ISSUE THREAD

**Issue by rqbanerjee** [eko/pihole-exporter#74]
Great project, thank you for publishing.
I have 
1. the exporter setup successfully, can 'curl 127.0.0.1:9617/metrics' and get back metrics
2. prometheus setup correctly, can hit the prometheus web dashboard and see pihole as a target
3. Installed the grafana dashboard for Pi-Hole Exporter, but it's 100% blank and missing data

I found this page that has a solution:
https://www.ncartron.org/grafana-and-pi-hole.html
That solution works.

And multiple people on the internet searching for the same solution:
here is one: https://www.gitmemory.com/issue/eko/pihole-exporter/3/495494348

Would you please add some of these dashboard installation instructions to the README in this repo? It would really ease usage of this great tool. Thanks for sharing it.

**Comment by JimKlapwijk** [comment]
It seems that the source is static. Had to rename everything in the JSON file by hand. I think if we can export something with a variable it's an easy fix. I've seen dashboards when you import them you have to give a source, and that would be ideal.
I don't have that much experience with exporting dashboards however.

Without hijacking your thread, I ran into an issue when you have multiple PiHole exporter that have data, it tries to diplay the data twice. 
I still need to fix that and then I was planning on creating a new template mentioning the thing above.

**Comment by JimKlapwijk** [comment]
[PI-Hole Exporter-1612862425130.json.zip](https://github.com/eko/pihole-exporter/files/5950206/PI-Hole.Exporter-1612862425130.json.zip)

Here we go, now supports multiple exports, and importing should go easier as it asks for the source :) please give it a try and let me know @rqbanerjee !

**Comment by SuperSandro2000** [comment]
The version on grafana.com is severely outdated. @eko could you update that one?

**Comment by Tom-jm03** [comment]
> The version on grafana.com is severely outdated. @eko could you update that one?

Yeah, please @eko update the one on grafana. Also the pie chart plugin has to be switched to the new one.

**Comment by eko** [comment]
Hi,

The dashboard has been updated with the one available in the GitHub repository (see revision number 3): https://grafana.com/grafana/dashboards/10176-pi-hole-exporter/?tab=revisions

I hope it helps.

I will try to upgrade it in the coming days.

**Comment by artur-borys** [comment]
Perhaps I can interest you in a dashboard I'm creating:

![obraz](https://github.com/eko/pihole-exporter/assets/48450925/3ec48f90-6639-4c08-b822-2a6a3137929f)
![obraz](https://github.com/eko/pihole-exporter/assets/48450925/6bb7d1df-e1bf-462c-9110-bd75c7aa1e95)
![obraz](https://github.com/eko/pihole-exporter/assets/48450925/3f641d6f-2080-4fb6-9861-8f75db09880d)

I'm open for suggestions about other charts, I've just created it today and it's my first dashboard created from scratch.

The dashboard has support for multiple instances, and by default displays for all, but one or multiple can be selected.

I'll share it some time next week and paste the link here
